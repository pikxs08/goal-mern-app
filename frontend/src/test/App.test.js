import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Spinner from "../components/Spinner";

test("renders Spinner component correctly", () => {
  const { container } = render(<Spinner />);
  expect(container.firstChild).toMatchSnapshot();
});
