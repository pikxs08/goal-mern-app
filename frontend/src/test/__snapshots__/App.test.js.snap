// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Spinner component correctly 1`] = `
<div
  class="loadingSpinnerContainer"
>
  <div
    class="loadingSpinner"
  />
</div>
`;
